### ISSUE TITLE
Request: Add Exceptions 

### ISSUE THREAD

**Issue by TrueGuardian32** [TheAlcanian/AdjustableAnything#3]
The ability to add exceptions to certain projectiles such as the flying piggy bank or the Glowsticks. Currently if Omnihome is on the PiggyBank will float an undetermined amount up going away from your character. 
If tile collision is off for ranged items the Glowsticks will go straight through the ground.

**Comment by TheAlcanian** [comment]
Will work on this. Thank you for your suggestion!

**Comment by TheAlcanian** [comment]
Will be implemented in the next version as an `Except special projectiles?` toggle. This toggle will except all projectiles of which would be bad to have all rules applied to them. I'm thinking about making an "except list" of every projectile in the game, and players can add to that list manually. This would be hard for me to implement, as I'm not sure that tMod has any "list" code for ModConfig.

**Comment by TrueGuardian32** [comment]
I don't think TMod does, but Hamstars helpers might help? It's a modcore https://forums.terraria.org/index.php?threads/mod-helpers.63670/
If you use this as a dependency it adds a bunch of useful code.

**Comment by TheAlcanian** [comment]
I'd rather keep this mod without dependencies, but thank you for your idea.

**Comment by TheAlcanian** [comment]
As it turns out, after surfing in ExampleMod's ExampleConfig.cs file, there is code to handle lists. That will be implemented in a future update, but for now this toggle should do.

**Comment by TrueGuardian32** [comment]
Alright, I'll try out the changes when their out an let you know what I think

**Comment by TrueGuardian32** [comment]
I went back over Modhelpers and remember that the following line was added, if you do decide to use the Helper mod this would help you use it without having it as a dependency. 

For modders
New to v1.2+ is the ability to enable github issues in-game. No mod dependency required!
To receive in-game issue reports from players: Your Mod overriding class must include the following properties (with your respective github user name and your mod's project name):
Code:
public static string GithubUserName => "hamstar0";
public static string GithubProjectName => "tml-hamstarhelpers-mod";
